utilities/tests/tree-test.metta

```diff
@@ -421,3 +421,30 @@
 !(assertEqual (treeComplexity (buildTree (AND (NOT A) (OR A B)))) 3)
 !(assertEqual (treeComplexity (mkNullVex (Cons (mkTree (mkNode AND) (Cons (mkTree (mkNode A) Nil) Nil)) Nil))) 0)
 !(assertEqual (treeComplexity (mkTree (mkNode OR) (Cons (mkTree (mkNode AND) (Cons (mkTree (mkNode A) Nil) Nil)) Nil))) 1)
+
+;; Test case for alphabetSize
+! (assertEqual (alphabetSize (mkTree (mkNode AND) (Cons (mkTree (mkNode A) Nil) (Cons (mkTree (mkNode B) Nil) Nil)))) 2 )
+! (assertEqual
+    (alphabetSize
+      (mkTree (mkNode OR) (Cons (mkTree (mkNode AND) (Cons (mkTree (mkNode X) Nil) (Cons (mkTree (mkNode Y) Nil) Nil)))
+          (Cons (mkTree (mkNode NOT) (Cons (mkTree (mkNode Z) Nil) Nil)) Nil))))
+      3)
+! (assertEqual (alphabetSize 
+      (mkTree (mkNode AND) (Cons (mkTree (mkNode OR) (Cons (mkTree (mkNode P) Nil) (Cons (mkTree (mkNode Q) Nil) Nil))) 
+          (Cons (mkTree (mkNode NOT) (Cons (mkTree (mkNode P) Nil) Nil)) Nil)))) 
+      2)
+
+;; Test case getComplexityCoef
```

Comment:
This test case belongs to the `cscore-test.metta` file since the function is defined there